modules/va_facilities/README.v1.yml

```diff
@@ -568,6 +814,168 @@ components:
           type: string
           format: date
           example: "2018-01-01"
+    DriveTimeBands:
+      description: JSON API-compliant object describing a VA facility and its drive time bands
+      allOf: 
+        - $ref: "#/components/schemas/Facility"  
+        - type: object
+          properties:
+            attributes:
+              type: object
+              required:
+                - drive_time_bands
+              properties:
+                drive_time_bands:
+                  type: array
+                  items:
+                    $ref: "#/components/schemas/DriveTimeBand"
+    DriveTimeBand:
+      description: a drive time band
+      type: object
+      properties:
+        name:
+          type: string
+          example: "drive_time_band_30"
```

Comment:
Does it add extra clarity to have a name string when number/unit convey roughly the same information?

Reply:
I think this is needed mostly for the first element in the array which is the lat/long of the facility itself. `{ value: 0, unit: "minutes" }` doesn't really tell you what that element is.